Automattic/jetpack
# Social | Initial state: Migrate useShareStatus feature flag to script data

Extracted out of #38952, this PR migrates on the `useShareStatus` feature flag and other feature flags will be migrated in the follow up PRs.






## Proposed changes:

* Migrate `useShareStatus` feature flag to script data

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* On Simple and Atomic sites, goto post editor
* Confirm that the editor loads fine
* Open Jetpack sidebar
* Confirm that share status UI is not visible
* On a Jetpack site, enable share status feature - `define( 'JETPACK_SOCIAL_HAS_SHARE_STATUS', true );`
* Goto post editor
* Open Jetpack/Social sidebar
* Confirm that share status UI is visible
* Publish a new post
* Confirm that share status UI is visible on post publish panel
* Turn the feature OFF
* Confirm that the UI is not visible
